aws_doc_sdk_examples_tools/doc_gen.py

```diff
@@ -5,26 +5,31 @@
 
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import Dict, List, Optional, Set
+from typing import Dict, Iterable, Optional, Set
 
 # from os import glob
 
-from aws_doc_sdk_examples_tools.metadata import Example, parse as parse_examples
-from aws_doc_sdk_examples_tools.metadata_errors import MetadataErrors
-from aws_doc_sdk_examples_tools.metadata_validator import validate_metadata
-from aws_doc_sdk_examples_tools.project_validator import (
+from .metadata import Example, parse as parse_examples
```

Comment:
Whoa, you can do this `.metadata` syntax? Docs?

Reply:
https://docs.python.org/3/reference/import.html#submodules